src/scripts/simple-multiple-choice.js

```diff
@@ -198,6 +200,16 @@ export default class SimpleMultiChoice extends H5P.EventDispatcher {
       return xApiGenerator.getResultPattern(this.state);
     };
 
+    /**
+     * Reset the task, resetting state and unchecking all checkboxes.
+     */
+    this.resetTask = function () {
+      this.state = this.state.map(option => ({ ...option, checked: false }));
+      this.$wrapper?.get(0)?.querySelectorAll('input[type=checkbox]').forEach(el => {
+        el.removeAttribute('checked');
+      });
```

Comment:
This content type can have either checkboxes or radio buttons, so this won't work with radio buttons. I think changing it to this should do the trick:
```suggestion
      this.$wrapper?.get(0)?.querySelectorAll('input[type=checkbox], input[type=radio]').forEach(el => {
        el.checked = false;
      });
```